src/components/Login/LoginFormJsonFile/LoginFormJsonFile.js

```diff
@@ -0,0 +1,65 @@
+
+import React from 'react';
+import { bool, string, func } from 'prop-types'; import { noop } from 'lodash';
+
+import ReactFileReader from 'react-file-reader'
+import Button from '../../Forms/Button';
+import styles from './LoginFormJsonFile.scss';
+
+export default class LoginFormJsonFile extends React.Component {
+  static propTypes = {
+    disabled: bool,
+    wif: string,
+    setWIF: func,
+    onLogin: func
+  };
+
+
+  static defaultProps = {
+    disabled: false,
+    wif: '',
+    setWIF: noop,
+    onLogin: noop
+  };
+
+
+handleFiles = files => {
+    const { setWIF } = this.props;
+
+    var reader = new FileReader();
+    reader.onload = e =>
+    {
+    let keyStoreJSON = JSON.parse(reader.result);
+
+    let accounts = keyStoreJSON.accounts.map(accounts =>
+      {
+        return accounts.key
+      });
+
+    setWIF(accounts[0]);

```

Comment:
A wallet can have more than one account in here.  I see you're just pulling the first account.  I think a better experience would be to let the user pick which account they wanted (the accounts have labels that we can display), and then login with that.

Reply:
I can do that but I need some clarification because, currently, the file generated when the  user creates an account via the client and saves the wallet file to its computer stays untouched afterwards.
In this case, there can only be 1 account in the wallet file. This is why I only check for the first value of the array as it should be the only value available.

However, maybe we can add the possibility for the user to save the account in an already existing wallet file. In this case, I can iterate through all the accounts in the wallet file and add the possibility for the user to select an account from all the accounts available in the wallet file.

What do you think of that?

Reply:
It's true that we're storing only a single account per wallet file at the moment.  However, the NEP6 standard permits multiple accounts per wallet file.  If someone tried to open a NEP6 file from Neon Wallet for example, we shouldn't just assume they exported one account from that app.

Reply:
We can also just load the first account for now, and an enhancement would be to let the user choose.  I'm okay with separating that work.  We should handle errors though (no accounts or JSON format is invalid).